mailbox/jpa/src/main/java/org/apache/james/mailbox/jpa/mail/JPAMessageMapper.java

```diff
@@ -299,10 +300,6 @@ private MessageMetaData copy(Mailbox mailbox, MessageUid uid, long modSeq, Mailb
         return save(mailbox, copy);
     }
 
-    /**
-     * @see org.apache.james.mailbox.store.mail.AbstractMessageMapper#save(Mailbox,
-     *      MailboxMessage)
-     */
```

Comment:
missing @override?

Reply:
Nope, this one could be private and does not override something.

Reply:
then you should not remove the comment?